Library/Homebrew/formula_creator.rb

```diff
@@ -9,11 +9,16 @@ module Homebrew
   #
   # @api private
   class FormulaCreator
-    attr_reader :args, :url, :sha256, :desc, :homepage
-    attr_accessor :name, :version, :tap, :mode, :license
-
-    def initialize(args)
-      @args = args
+    attr_reader :url, :sha256, :desc, :homepage
+    attr_accessor :name, :tap, :mode
```

Comment:
Could move `fc.mode` assignment up into a variable `mode =` and also pass it to `initialize` and do this, I think:

```suggestion
    attr_reader :url, :sha256, :desc, :homepage, :tap, :mode
    attr_accessor :name
```

Reply:
Removed `mode` from accessors.